### DISCUSSION TITLE
Fiserv Core Support

### DISCUSSION THREAD

**Discussion by tranqy** [Fiserv/Support#77] [Q&A]
We are very interested in using these API's to connect to Fiserv cores. We already have a number of file exchanges with Fiserv banks, and would like to start using APIs where appropriate. 

Which cores does this API currently support? We have customers on Premier, and DNA, and we have some in the pipeline on other Fiserv cores. 

We think the capabilities on the API cover what we need, we're also curious what variations we might see across cores in the API?

Thank you for making Fiserv cores more accessible to the ecosystem!

**Comment by TaniaS-fiserv** [comment]
Hi @tranqy Thank you for reaching out to us !!

@bobburghardt  @tsegurajimen could you please assist @tranqy with his queries.

Thanks!

**Reply by bobburghardt** [reply]
@tranqy - Presently we have selected our initial set of FinTech partners to work with us on the Developer Studio.  If you supply us with your name, your company name, and contact information, we can pass that information on to the team evaluating the next set of partners.  Please note that that activity will occur early in 2023.
